app/views/ncr/_header_links_next.html.haml

```diff
@@ -1,5 +1,5 @@
-%li
-  = link_to "New request", new_ncr_work_order_path, class: "new-request-button"
+%li{class: "new-request-button #{show_beta_new_request? ? 'active' : ''}"}
```

Comment:
Yes, @rememberlenny I'm pretty sure I'm applying an `active` class at the correct time. I'm wondering if you can help with the implementation of the color change itself because you wrote the `active` code for the other buttons here:

https://github.com/18F/C2/blob/master/app/assets/stylesheets/redesign/_sidebar-nav.scss.erb#L34-L55

and here:

https://github.com/18F/C2/blob/master/app/assets/stylesheets/redesign/_sidebar-nav.scss.erb#L184-L226

and I don't how to cleanly add the handling in for this particular button in the sidebar.